Tools/CheckCmdletXmlDocs.psm1

```diff
@@ -116,15 +108,33 @@ function Check-CmdletDoc() {
         "GcSql" = "Google Cloud SQL"
     }
 
-    $numPassedCmdlets = 0
+    # If the CmdletNames parameter is specified, get the cmdlets explicitly named.
+    if ($CmdletNames) {
+         $cmdlets = GetCmdletsByName $cmdletNames $cmdlets
+    }
+
+    # If the CloudProducts parameter is specified, get the cmdlets in the named products. 
+    if ($CloudProducts) {
+        $cmdlets = GetCmdletsByProduct $CloudProducts $apiMappings $cmdlets
+    }
+
+    # Get the cmdlets that are whitelisted for (don't need a) OutputType
```

Comment:
How about:

# Cmdlets can be whitelisted for valid cases. For example, it they intentionally don't
# produce any output we should not warn because it does not have an OutputType specified.